History.md

```diff
@@ -18,6 +18,15 @@
   * Add license to gemspec (Jordi Massaguer Pla #72, Anatol Pomozov #89,
     Anatol Pomozov #90, Johannes Barre #109, Mandaryn #115,
     Chris Marshall #120, Robert Reiz #126, Nic Benders #127, Nic Benders #130)
+  * Feature/bulk pr for readme updates (Patrick Mulder #65,
+    Troels Knak-Nielsen #74, Jeff Casimir #76)
+  * Switches README format to markdown (Dennis Sivia #176)
+  * Converts History.txt to markdown (Dennis Sivia #179)
```

Comment:
Thanks for the review.
Updated as 1 more commit.
After finishing review, we can squash it.